docs/source/top_level_elements/genre.rst

```diff
@@ -0,0 +1,164 @@
+
+.. code-block:: xml
+
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/genreForms/gf2014026131">Newsletters</genre>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+    <https://example.org/object/1> edm:hasType <http://id.loc.gov/authorities/genreForms/gf2014026131> .
+
+In 124 of the records in the VP Moore collection, with `@authority='lcgft'`, the `@authorityURI = 'http://id.loc.gov/authorities/genreForms'` is used; e.g. `vpmoore:50 <https://digital.lib.utk.edu/collections/islandora/object/vpmoore:50/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="lcgft" authorityURI="http://id.loc.gov/authorities/genreForms" valueURI="http://id.loc.gov/authorities/genreForms/gf2014026173">Scrapbooks</genre>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+    <https://example.org/object/1> edm:hasType <http://id.loc.gov/authorities/genreForms/gf2014026173> .
+
+genre[@authority = 'lcmpt']
+---------------------------
+
+`@authority = 'lcmpt'` is used in the Van Vactor collection to express genre/performance medium instrumentation information; e.g. `vanvactor:12350 <https://digital.lib.utk.edu/collections/islandora/object/vanvactor:12350/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015074">bassoon</genre>
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015342">horn</genre>
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015748">trumpet</genre>
+    <genre authority="lcmpt" valueURI="http://id.loc.gov/authorities/performanceMediums/mp2013015540">percussion</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/genreForms/gf2014027156">Variations (Music)</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/genreForms/gf2014026956">Musical sketches</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/genreForms/gf2014026097">Excerpts</genre>
+    <genre authority="lcgft" valueURI="http://id.loc.gov/authorities/subjects/sh99001779">Scores</genre>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+    @prefix gnd: <https://d-nb.info/standards/elementset/gnd#> .
+
+    <https://example.org/object/1>
+        gnd:playedInstrument <http://id.loc.gov/authorities/performanceMediums/mp2013015074> ;
+        gnd:playedInstrument <http://id.loc.gov/authorities/performanceMediums/mp2013015342> ;
+        gnd:playedInstrument <http://id.loc.gov/authorities/performanceMediums/mp2013015748> ;
+        gnd:playedInstrument <http://id.loc.gov/authorities/performanceMediums/mp2013015540> ;
```

Comment:
This won't work in my opinion.  There are a few reasons why, but let's start by looking at our property.

https://d-nb.info/standards/elementset/gnd#playedInstrument has a `rdfs:range` of https://d-nb.info/standards/elementset/gnd#SubjectHeading and a `rdfs:domain` of https://d-nb.info/standards/elementset/gnd#DifferentiatedPerson.  

1. Technically, we have an issue with range here.  The object of our triple is a http://www.loc.gov/mads/rdf/v1#Authority, http://www.loc.gov/mads/rdf/v1#Medium, and http://www.w3.org/2004/02/skos/core#Concept. These inherit types like http://www.w3.org/2002/07/owl#Class, but not https://d-nb.info/standards/elementset/gnd#SubjectHeading.  I'm not too worried about this though because we've violated it (see our use of edm:), libraries violate it all the time, and the nature our the object of our triple is a "subject heading."
2. I'm more concerned with the latter. This property expects to be used by a rdfs:Class (our fedora object) that is a https://d-nb.info/standards/elementset/gnd#DifferentiatedPerson.  Technically, we have the same problem as above, but I don't think we had a "differentiated person" or a "person."  We have a digital object.  

Finally, I don't think we should go looking for an alternative here.  Let's just keep it simple and use the same properties we're already using here.

Reply:
@markpbaggett thanks for your thoughts here. In the process of reviewing a different section of the mapping document, I realized we should be sticking with `opaquenamespace` for these. I will update this shortly.